cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/send_xcm.rs

```diff
@@ -16,7 +16,7 @@
 use crate::*;
 
 #[test]
-fn send_xcm_from_rococo_relay_to_westend_asset_hub() {
+fn send_xcm_from_rococo_relay_to_westend_asset_hub_should_fail_on_not_applicable() {
```

Comment:
@bkontur can we add a new test here where we also verify `send_xcm` when there is an opened lane?

Aka open a lane and run+verify previous version of the test.